### ISSUE TITLE
Server not reachable

### ISSUE THREAD

**Issue by CrashedCode** [Hopson97/open-builder#78]
# Bug Name
Engine exited with error: "Connection to server could not be established"
## Describe the bug
The game doesn't start at all.

## To Reproduce
Just start the run.sh file

## Expected behavior
I expected the game to start
## Screenshots
![Bildschirmfoto 2020-01-18 03:02:55](https://user-images.githubusercontent.com/60023210/72656941-43e6bd80-399f-11ea-993d-6541c493d1e7.png)

## Desktop (please complete the following information):
 - OS: Linux Kali 2019.4

## Additional context
It's a complete fresh install of everything

**Comment by Hopson97** [comment]
This is because of the issue described in #77. 

There are 3 work around for the time being:

## Build and run in release modd

This will make the world generation faster, meaning the server will actually be ready before the client starts.

```
sh scripts/build.sh release
sh scripts/run.sh release
```

## Run the server and the client separately 

```
sh scripts/run.sh -server 4

#wait a few secs, then on another terminal

sh scripts/run.sh -client x
```
## Edit the code a bit to increase the delay 

In main.cpp there is a function called `launchBoth`

This is the function that will launch the server on one thread, wait about 500ms, and then launch the client. 

You can increase the number in the sleep_for function call to mitigate the issue.

**Comment by Hopson97** [comment]
Solved by PR #79
